source/usage-examples/convenient-transactions.txt

```diff
@@ -0,0 +1,174 @@
+.. _node-usage-convenient-txn:
+
+================================
+Perform a Convenient Transaction
+================================
+
+The {+driver-short+} provides two APIs to perform transactions, the Core
+API and the **Convenient Transaction API**. By using the Convenient Transaction
+API, you can perform a transaction without being responsible for
+ensuring the transaction was committed or aborted. This API
+automatically incorporates error handling logic for certain error types,
+such as ``TransientTransactionError`` and ``UnknownTransactionCommitResult``.
+
+This usage example provides sample code that uses the Convenient
+Transaction API. To learn more about the Core API and view an example,
+see the :ref:`nodejs-transactions` guide.
+
+.. tip::
+   
+   To learn more about the ``TransientTransactionError`` error, see the
+   :manual:`TransientTransactionError description
+   </core/transactions-in-applications/#transienttransactionerror>` on
+   the Driver API page in the Server manual.
+
+   To learn more about the ``UnknownTransactionCommitResult`` error, see the
+   :manual:`UnknownTransactionCommitResult description
+   </core/transactions-in-applications/#unknowntransactioncommitresult>` on
+   the Driver API page in the Server manual.
+
+The {+driver-short+} provides the following helper methods to perform a
+transaction by using the Convenient Transaction API:
+
+- ``withSession()``: Runs a given operation within a session. You do not
+  have to handle the creation or termination of the session.
+- ``withTransaction()``: Starts a transaction and runs a provided
+  function. This method ensures that the ``commitTransaction()`` method
+  is called when all of the operations in the function complete.
+
+.. important:: Return Types and Values
+   
+   Starting in version 6.0 of the {+driver-short+}, these methods return
+   the value that the provided function returns.
```

Comment:
Issue:
I think shortly after you prepared this PR, we had a team discussion and decision to exclude this versioned information. 

Question:
Is the Convenient Transaction API new to some other version other than v6? Was there a previous behavior to compare it with? Or are these methods not specific to the Convenient Transaction API in which case they probably shouldn't be described as such in the previous list.

Reply:
I can remove the versioning from this sentence - the previous behavior is described in the what;s new/upgrade pages.
These methods are specific to the this API.